.github/workflows/linters.yml

```diff
@@ -5,7 +5,7 @@ on: pull_request
 env:
   FORCE_COLOR: 1
 
-jobs:
+jobs: 
   lighthouse:
     name: Lighthouse
     runs-on: ubuntu-22.04
```

Comment:
- [ ] Please set up the linters correctly so the github can perform linter checks on your repository. Your linters were set up properly in your previous branch the only problem was your stylint checks not passing. To pass your stylint either you need to correct the errors manually or you can automatically correct most of them using command 
```sh
 npx stylelint "**/*.{css,scss}" --fix  
```